### DISCUSSION TITLE
如何配置管理页面的外网访问

### DISCUSSION THREAD

**Discussion by liaobadadi** [vernesong/OpenClash#2648] [Q&A]
lastversion ：v0.45.51-beta
家里移动宽带，路由器安装了Openclash,ddns 了路由器的IPV6地址， 在外网可以通过IP或者域名访问家里路由器，
![image](https://user-images.githubusercontent.com/61466838/185768769-ad6ec966-5813-4b57-a34a-7b186c38bb78.png)

但是在外网无法打开OPENCLASH的管理页面YACD ,DASHBOARD.不知道如何设置。

-目前打开YACD ,填api url ，内网ip-http://192.168.1.1 不对，127.0.0.1:9090 也不对，密码123456
![image](https://user-images.githubusercontent.com/61466838/185768814-1df1724c-2f71-45ed-adfe-d980bbeeb43d.png)

-Dashboard 也是一样
![image](https://user-images.githubusercontent.com/61466838/185768863-72d713a4-2b69-474d-9622-dffe9a44fada.png)

懂得大佬帮帮解答下~！， 感谢，跪了！

**Comment by liaobadadi** [comment]
@vernesong  ,could you help ?  thanks！@ @

**Comment by vernesong** [comment]
映射9090端口，登录填外网实际访问的地址和端口
